streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -742,6 +756,30 @@ public void onRestoreSuspended(final TopicPartition topicPartition, final String
         }
     }
 
+    final class DelegatingStandbyUpdateListener implements StandbyUpdateListener {
+
+        @Override
+        public void onUpdateStart(final TopicPartition topicPartition, final String storeName, final long startingOffset, final long currentEndOffset) {
```

Comment:
nit: these method signatures are all really long, can you break up the parameters? This is the format we use:
```suggestion
public void onUpdateStart(final TopicPartition topicPartition, 
                          final String storeName, 
                          final long startingOffset, 
                          final long currentEndOffset) {
```